### PR TITLE
{bio}[GCC/12.3.0] PSASS v3.1.0

### PR DESCRIPTION
**Pull request by WilleBell**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/PSASS/PSASS-3.1.0-GCC-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+dependencies = [
+    ('HTSlib', '1.18'),
+]
+
+source_urls = ['https://github.com/SexGenomicsToolkit/PSASS/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['6c22fb2db52df76bdf8e5073895da11678a35c4a0e1500245e19d13c1f67df2b']
+
+# just run make in the install dir
+skipsteps = ['configure', 'install']
```
Comment (casparvl) [inline easybuild/easyconfigs/p/PSASS/PSASS-3.1.0-GCC-12.3.0.eb:25]: While this probably works, I'm wondering: isn't the `MakeCp` EasyBlock a better fit here? https://docs.easybuild.io/version-specific/generic-easyblocks/

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
vera-skylake-build - Linux Rocky Linux 8.9, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/Micket/3315ece0a9d4687d5276aa41877397de for a full test report.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
@boegelbot please test @ jsc-zen3

**Pull request merged**